.gitignore

```
@@ -65,4 +65,5 @@ CMakeLists.txt.user
 install
 
 # language server protocol
-.clangd
\ No newline at end of file
+.clangd.DS_Store
```

已经被git追踪的文件再被添加到.gitignore中是无效的，手动删除.DS_Store后，重新push这条删除文件的commit

这一行改回来 
```suggestion
.clangd
```